nixos/doc/manual/release-notes/rl-2205.section.md

```diff
@@ -656,6 +656,15 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - The `services.stubby` module was converted to a [settings-style](https://github.com/NixOS/rfcs/blob/master/rfcs/0042-config-option.md) configuration.
 
+- The option `services.xserver.desktopManager.runXdgAutostartIfNone` was added in order to
+  automatically run XDG autostart files for sessions without a desktop manager.
+  This replaces helpers like the `dex` package.
+
+- When setting `i18n.inputMethod.enavled` to `fcitx5`, it no longer creates coresponding systemd user services.
```

Comment:
Typo: `i18n.inputMethod.enabled`.

Reply:
Also, could you change all those literal to the link to the option? For example:

```
`i18n.inputMethod.enabled` -> [i18n.inputMethod.enabled](#opt-i18n.inputMethod.enabled)
```

This will make a link to the manual that will allow the users to click on it.

Reply:
> Also, could you change all those literal to the link to the option?

Seems it's not enforced in the file. There is only little usage of this syntax.

Reply:
> Seems it's not enforced in the file. There is only little usage of this syntax.

Still would be better since this makes it easier to switch between the release notes/option description. But your choice.

Anyway, please fix at least the typo.

Reply:
Fixed.